### PR TITLE
API controllers V1

### PR DESCRIPTION
**Pull request by isratrade**

### CODE REVIEW

**Comment by ohadlevy** [comment]
@pitr-ch mind having a look to see if it breaks anything for you?

thanks!

File: app/models/hostgroup.rb [hunk 1]
```diff
@@ -61,7 +61,7 @@ def all_puppetclasses
   end
 
   def to_label
-    return unless name
+    ret::urn unless name
```
Comment (pitr-ch) [inline app/models/hostgroup.rb:64]: Is it a typo?

**Comment by skottler** [comment]
test <- trying out the new automated PR system

File: app/models/hostgroup.rb (hunk 1 above)
Comment (isratrade) [inline app/models/hostgroup.rb:64]: Yes indeed, is a typo. Thanks @pitr-ch

**Comment by pitr-ch** [comment]
@ohadlevy pulling and checking with Katello today

**Comment by pitr-ch** [comment]
looks ok for Katello, ACK, Thanks!

**Comment by ohadlevy** [comment]
closed, will merge #239 instead

**Pull request closed without merging**